### ISSUE TITLE
Are you considering continuing to iterate on this project?

### ISSUE THREAD

**Issue by yubing744** [victoryang00/move-fmt#1]
There are now two Issues that require the move-fmt function. Hope you can continue to iterate on this project, or donate this project to the move community.

https://github.com/move-language/move/issues/166
https://github.com/move-language/move/issues/257

**Comment by vickiegpt** [comment]
Yep for donating it, but the project has some bugs for parsing associativity. I'm considering rewriting some of the code emissions.

**Comment by yubing744** [comment]
thanks.
